code/datums/elements/footstep_override.dm

```diff
@@ -0,0 +1,59 @@
+/datum/element/footstep_override
+	element_flags = ELEMENT_BESPOKE|ELEMENT_DETACH_ON_HOST_DESTROY
+	argument_hash_start_idx = 2
+	var/clawfootstep
+	var/barefootstep
+	var/heavyfootstep
+	var/footstep
+	var/priority
+	var/list/occupied_turfs = list()
```

Comment:
The names are pretty self-explanatory but please document these for our metrics.
And the procs while you are at it.